### DISCUSSION TITLE
Team Page -> Galley Page

### DISCUSSION THREAD

**Discussion by lazarusA** [vuejs/vitepress#2835] [Q&A] [answered]
Any suggestions on how to create a Gallery page of images? At the moment I'm going for a Team Page, using the `avatar` attribute to include the image, however is not optimal and the image is really small, I haven't found a way to make it bigger. Any ideas would be greatly appreciated. Thanks.

**Comment by brc-dd** [comment]
You can use vue components in any markdown page. Write a custom one to render a gallery of images.

**Reply by lazarusA** [reply]
Apologies for the dummy questions, but I'm new to this. Let's say  we have this file structure https://vitepress.dev/guide/getting-started#file-structure, where do I define my new `component`, how do I called it in `index.md` for example? 

Even some links on how this can be done in a simple way would help. But, all things around look overcomplicated.

**Reply by brc-dd** [reply]
https://vitepress.dev/guide/using-vue#using-components You can store it anywhere you want. You'll need to update your import path accordingly.

**Reply by brc-dd** [reply, marked as answer]
Refer this as example: https://stackblitz.com/edit/vite-asrc1a?file=docs/index.md,docs/components/GalleryImage.vue,docs/components/Gallery.vue

**Reply by lazarusA** [reply]
This is great, thanks a lot. I will take a deep look at this and move forward. Thanks again.
